src/picongpu/include/fields/laserProfiles/laserGaussianBeam.hpp

```diff
@@ -129,7 +129,7 @@ namespace picongpu
             return elong;
 #else
             //beam waist in the near field: w_y(y=0) == W0
-            const float_X w_y = W0 * sqrtf( float_X(1.0) + ( FOCUS_POS / y_R )*( FOCUS_POS / y_R ) );
+            const float_X w_y = W0 * algorithms::math::sqrt( float_X(1.0) + ( FOCUS_POS / y_R )*( FOCUS_POS / y_R ) );
             //! the Gouy phase shift
             const float_X xi_y = atanf( -FOCUS_POS / y_R );
```

Comment:
@psychocoderHPC Should I know add this in another pull request (as discussed *offline*  3 days ago) or should I add this to this pull request?